source/back.tex

```diff
@@ -6,6 +6,18 @@ \chapter{Bibliography}
 
```

Comment:
The statement on the previous line isn't true; we don't cite some of these documents informatively. ISO/CS wanted that statement removed, so let's do that at the same time.

Reply:
Removed.